### ISSUE TITLE
Mass Obsoletion: focus on branch 'integumentary system disorder' (MONDO:0002051)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6749]
branch:  'integumentary system disorder' (MONDO:0002051)
Terms to obsolete: 
obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0016434 | acquired dermis elastic tissue disorder
http://purl.obolibrary.org/obo/MONDO_0019271 | acrokeratoderma
http://purl.obolibrary.org/obo/MONDO_0015818 | aggressive primary cutaneous B-cell lymphoma
http://purl.obolibrary.org/obo/MONDO_0015817 | aggressive primary cutaneous T-cell lymphoma
http://purl.obolibrary.org/obo/MONDO_0017670 | autosomal dominant diffuse mutilating palmoplantar keratoderma
http://purl.obolibrary.org/obo/MONDO_0020093 | autosomal dominant isolated diffuse palmoplantar keratoderma
http://purl.obolibrary.org/obo/MONDO_0017270 | autosomal ichthyosis syndrome
http://purl.obolibrary.org/obo/MONDO_0020096 | autosomal recessive isolated diffuse palmoplantar keratoderma
http://purl.obolibrary.org/obo/MONDO_0017922 | deafness-onychodystrophy syndrome
http://purl.obolibrary.org/obo/MONDO_0019292 | dermis elastic tissue disorder
http://purl.obolibrary.org/obo/MONDO_0020197 | EEC syndrome and related syndrome
http://purl.obolibrary.org/obo/MONDO_0019277 | epidermal appendage anomaly
http://purl.obolibrary.org/obo/MONDO_0035685 | epidermolysis bullosa simplex with extracutaneous involvement
http://purl.obolibrary.org/obo/MONDO_0035684 | epidermolysis bullosa simplex without extracutaneous involvement
http://purl.obolibrary.org/obo/MONDO_0021034 | hereditary alopecia
http://purl.obolibrary.org/obo/MONDO_0033947 | hereditary angioedema with normal C1Inh
http://purl.obolibrary.org/obo/MONDO_0957001 | hereditary mixed dermis disorder
http://purl.obolibrary.org/obo/MONDO_0017263 | inherited ichthyosis syndromic form
http://purl.obolibrary.org/obo/MONDO_0017262 | inherited non-syndromic ichthyosis
http://purl.obolibrary.org/obo/MONDO_0015950 | inherited skin tumor
http://purl.obolibrary.org/obo/MONDO_0017667 | isolated diffuse palmoplantar keratoderma
http://purl.obolibrary.org/obo/MONDO_0017673 | isolated focal palmoplantar keratoderma
http://purl.obolibrary.org/obo/MONDO_0019281 | isolated genetic hair shaft abnormality
http://purl.obolibrary.org/obo/MONDO_0016518 | isolated punctate palmoplantar keratoderma
http://purl.obolibrary.org/obo/MONDO_0017676 | marginal papular palmoplantar keratoderma
http://purl.obolibrary.org/obo/MONDO_0020181 | mesenchymatous palpebral tumor
http://purl.obolibrary.org/obo/MONDO_0020178 | palpebral lentiginosis
http://purl.obolibrary.org/obo/MONDO_0020180 | palpebral piliary tumor
http://purl.obolibrary.org/obo/MONDO_0020176 | palpebral sebaceous gland tumor
http://purl.obolibrary.org/obo/MONDO_0020177 | pigmented palpebral tumor
http://purl.obolibrary.org/obo/MONDO_0020174 | precancerous lesion of palpebral epidermis
http://purl.obolibrary.org/obo/MONDO_0019599 | primary lipodystrophy
http://purl.obolibrary.org/obo/MONDO_0019286 | sebaceous gland anomaly
http://purl.obolibrary.org/obo/MONDO_0018745 | superficial pemphigus
http://purl.obolibrary.org/obo/MONDO_0019176 | trichorhinophalangeal syndrome type I or III

**Comment by nicolevasilevsky** [comment]
[spreadsheet](https://docs.google.com/spreadsheets/d/1ktPh8VubQdOssrZD3_H-VXcM5OvziOPbeYAFFu1EcVg/edit#gid=113101891)

**Comment by nicolevasilevsky** [comment]
the entire spreadsheet is ready for review now

**Comment by katiermullen** [comment]
@nicolevasilevsky I reviewed this branch and left just a few questions for you :)

**Comment by nicolevasilevsky** [comment]
Thanks @katiermullen!

This should be ready for mass obsoletion now.

**Comment by nicolevasilevsky** [comment]
@katiermullen I updated some of the superclass assertions based on your comments, thank you!

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1dKkp-rH-pmcIfm-nYz34tAy1vlAtZsnf/edit#gid=1660849274 


TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab) !!!!1 term comment will need to be added manually
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)

Note: Once the to-do list is done, 22 terms for which reviewers disagreed will remain to be assessed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
